odoo/addons/base/views/res_country_views.xml

```diff
@@ -11,6 +11,7 @@
                 <tree string="Country">
                     <field name="name"/>
                     <field name="code"/>
+                    <field name="image_url" widget="image_url" options="{'size': [50, 50]}"/>
```

Comment:
Having the flag in the tree view is "cool" but I'm not sure of the added value.
Especially considering that it makes the row height variable which is a bit meh:

![image](https://user-images.githubusercontent.com/41282536/91956683-55e65800-ed05-11ea-9819-556fabcf77db.png)

Reply:
Indeed in tree view, not sure this is useful. Form view should be sufficient.